process-engine/bpmn-studio
# Rework Event System of Property Panel

## What did you change?

This branch reworks the event handling in the property panel. Instead of letting each section subscribe itself to the events, the property panel subscribes itself and passes the found element to the sections. The sections just check if they can handle the element.

## How can others test the changes?

- Checkout branch & start bpmn-studio
- Create a process & start by editing it (a lot)

Now you can see that the "old" features work like they should while the code is much cleaner.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
